### DISCUSSION TITLE
使用app1.10版本出现白屏

### DISCUSSION THREAD

**Discussion by walking-blockhead** [lencx/ChatGPT#1086] [Q&A]
我下载了chatGPT1.10，升级后发现进入就是白屏，目前我只能用网页版使用了，所以我的网络应该是没有问题的

我的系统是mac m2 系统14.0，您有时间方便的话，帮忙看下，谢谢。

[白屏1](https://i.postimg.cc/JzjHxcSx/1.png)

[白屏2](https://i.postimg.cc/85cC2hXq/2.png)

**Comment by shilohzhen** [comment]
我也是，我是今天刚出现的。白屏。
